Documentation/criu.txt

```diff
@@ -120,10 +124,16 @@ The following levels are available:
             run to unlock network in a target network namespace
 
         *setup-namespaces*:::
-            run once root task just been created
+            run once root task has just been created
             with required namespaces. Note it is an early stage
-            of restore, when nothing is restored yet except for namespaces
-            themselves
+            of restore, when nothing is restored yet, except for
+            namespaces themselves
+
+        *post-setup-namespaces*:::
+            called after the namespaces are configured
+
+        *orphan-pts-master*:::
+            called after master pty is opened and unlocked
```

Comment:
This one is only for the rpc mode and contains a file descriptor which is a master pty.